app-modules/alert/src/Filament/Resources/AlertResource/Pages/ManageAlerts.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Assist\Alert\Filament\Resources\AlertResource\Pages;
+
+use Filament\Forms\Form;
+use Filament\Tables\Table;
+use Filament\Infolists\Infolist;
+use Assist\Alert\Enums\AlertStatus;
+use Assist\Alert\Enums\AlertSeverity;
+use Filament\Forms\Components\Select;
+use Illuminate\Support\Facades\Cache;
+use Filament\Forms\Components\Textarea;
+use Filament\Tables\Actions\EditAction;
+use Filament\Tables\Actions\ViewAction;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Actions\CreateAction;
+use Filament\Tables\Actions\DeleteAction;
+use Filament\Tables\Filters\SelectFilter;
+use Assist\Alert\Models\Contracts\Alertable;
+use Filament\Infolists\Components\TextEntry;
+use Filament\Tables\Actions\BulkActionGroup;
+use Filament\Tables\Actions\DeleteBulkAction;
+use Filament\Resources\Pages\ManageRelatedRecords;
+
+class ManageAlerts extends ManageRelatedRecords
+{
+    protected static string $relationship = 'alerts';
+
+    // TODO: Automatically set from Filament based on relationship name
+    protected static ?string $navigationLabel = 'Alerts';
+
+    // TODO: Automatically set from Filament based on relationship name
+    protected static ?string $breadcrumb = 'Alerts';
+
+    protected static ?string $navigationIcon = 'heroicon-o-bell-alert';
+
+    public static function getNavigationItems(array $urlParameters = []): array
+    {
+        $item = parent::getNavigationItems($urlParameters)[0];
+
+        $ownerRecord = $urlParameters['record'];
+
+        /** @var Alertable $ownerRecord */
+        $alertsCount = Cache::tags('alert-count')
+            ->remember(
+                "alert-count-{$ownerRecord->getKey()}",
+                now()->addMinutes(5),
+                function () use ($ownerRecord): int {
+                    // TODO: When it is decided how alerts are "resolved" this will need to take that into account to only display unresolved alerts
+                    return $ownerRecord->alerts()->status(AlertStatus::Active)->count();
```

Comment:
I agree, I'll remove it. Also I only refactored this because I noticed that Prospect Alerts weren't implemented despite having the subnavigation tab. So I looked around and saw several were either not implemented or duplicated and figured it would better to finish all of them. In addition, none of them seemed to have any specific logic that differed between Student and Prospect. I can undo them if you want, but I also don't feel like it's too much of a burden if we want to fork one in the future.